src/rofl/common/ctimer.hpp

```diff
@@ -92,6 +86,8 @@ class ctimer {
    */
   const ctimespec &get_tspec() const { return tspec; };
 
+  cthread_timeout_event *get_callback() { return e; }
```

Comment:
can't be constant otherwise callbacks and their implementations should be constant as well which shouldn't be the case